config.py

```diff
@@ -169,11 +169,11 @@ def create_config(config_file=None):
 # Display - Inky pHAT
 elif "inkyphat" in conf["atm"]["display"]:
     try:
-        from inky import InkyPHAT
+        from inky.auto import auto
 
         WHITE = 0
         BLACK = 1
-        INKY = InkyPHAT("black")
+        INKY = auto(ask_user=True, verbose=True)
```

Comment:
What is mean with ask_user? Any user interactions needed?

Reply:
no interaction needed if the board gets detected, if detection does not work then user can choose options.
looking at the library again it could be even better to omit, as it's false per default: https://github.com/pimoroni/inky/blob/master/library/inky/auto.py